lib/DB/Schema/ResultSet/User.pm

```diff
@@ -272,9 +271,14 @@ or an arrayref of C<DBIx::Class::ResultSet::User>
 
 sub getCourseUsers ($self, %args) {
 	my $course       = $self->rs('Course')->getCourse(info => getCourseInfo($args{info}), as_result_set => 1);
-	my @course_users = $self->rs('CourseUser')->search({
-		'course_id' => $course->course_id
-	});
+	my @course_users = $self->rs('CourseUser')->search(
+		{
+			'course_id' => $course->course_id
+		},
+		{
+			prefetch => [qw/role/]
+		}
+	);
```

Comment:
Why is this so spread out?  It can be done in two lines if you change it to
```suggestion
	my @course_users =
		$self->rs('CourseUser')->search({ 'course_id' => $course->course_id }, { prefetch => [qw/role/] });

```
pertidy accepts that.  The same can be done on lines 337 - 344 below.